spec/models/user_spec.rb

```diff
@@ -59,6 +59,7 @@
       create(:donor, admin: true, name: "Admin/Donor")
       create(:admin, name: "Admin")
       create(:cyclist, name: "Cyclist")
+      create(:donation, donor: donor)
```

Comment:
@jferris right, sorry for pulling you into this.

A `Donation` has a `has_one :donor, through: :location` relationship. The factory should be called with a `location`, not a `donor`.